### PR TITLE
Add hex.pm and documentation badge

### PR DESCRIPTION
**Pull request by dweinstein**
Make it easier for new users to know the current version and find the documentation using two new badges.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ExAws
 =====
+[![Hex.pm](https://img.shields.io/hexpm/v/ex_aws.svg)](https://hex.pm/packages/ex_aws)
+[![Build Docs](https://img.shields.io/badge/documentation-v1.1.3-blue.svg)](https://hexdocs.pm/ex_aws/ExAws.html)
```
Comment (ericmj) [inline README.md]: It will be easy to forget to bump this version when new releases are made.

Comment (dweinstein) [inline README.md]: ok I can go with "latest" or "release" -- preference?

Comment (dweinstein) [inline README.md]: I went with `hexdocs|release`

**Comment by dweinstein** [comment]
Let me know what you think now

**Comment by benwilson512** [comment]
Thanks!

**Pull request merged**